packages/suite/src/utils/suite/guide.ts

```diff
@@ -0,0 +1,40 @@
+import { GuideCategoryType, GuideItemType } from '@suite-types/guide';
+
+export const getItemTitle = (item: GuideItemType, language: string): string =>
+    item.title[language || 'en'];
+
+export const getNodeById = (id: string, root: GuideItemType): GuideItemType | undefined => {
+    if (id === root.id) {
+        return root;
+    }
+
+    if (root.type === 'page') {
+        return undefined;
+    }
+
+    return root.children.map(child => getNodeById(id, child)).find(it => it !== undefined);
+};
+
+export const getAncestorIds = (id: string): string[] =>
+    id
+        .split('/')
+        // omit the item itself - only consider its ancestors
+        .slice(0, -1)
+        // 'Absolutize' each exploded parent, effectively getting its ID.
+        .reduce<string[]>((ids, ancestor) => {
+            const id = `${ids[ids.length - 1] || '/'}${ids.length > 1 ? '/' : ''}${ancestor}`;
+
+            return [...ids, id];
+        }, [])
+        .filter(id => id !== '/');
```

Comment:
nit: we could handle the trivial case and exit early instead of filtering later on.
```suggestion
export const getAncestorIds = (id: string): string[] => {
    if (id === '/') {
        return [];
    }
    return id
        .split('/')
        // omit the item itself - only consider its ancestors
        .slice(0, -1)
        // 'Absolutize' each exploded parent, effectively getting its ID.
        .reduce<string[]>((ids, ancestor) => {
            const id = `${ids[ids.length - 1] || '/'}${ids.length > 1 ? '/' : ''}${ancestor}`;

            return [...ids, id];
        }, []);
}